nexum_core/benches/README.md

```
@@ -1,215 +1,215 @@
+Focuses on WHERE clause evaluation performance:
+
+- **Simple Comparisons**: Basic equality, inequality operations
+- **Complex Expressions**: AND, OR, nested conditions
+- **LIKE Patterns**: Pattern matching with wildcards
+- **IN Lists**: Performance with different list sizes
+- **BETWEEN Ranges**: Range-based filtering
+- **Batch Evaluation**: Filter performance across large datasets
+
+## Running Benchmarks
+
+### Run All Benchmarks
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)